vagrant/debian/README.md

```diff
@@ -15,11 +15,6 @@ $ sudo apt-get install nfs-kernel-server
 $ sudo apt-get install vagrant
 ```
 
-4. Install vagrant-reload plugin (https://github.com/aidanns/vagrant-reload/tree/master)
```

Comment:
Correct, to stop using the reload plugin, you need to add the `, reboot: true` argument to the shell provisioner and remove the reload line. You had this in the past, but you changed it for the reload plugin. I suppose because of issues with libvirt. It's not needed for VB though, so I only reverted it for VB.